playbooks/roles/nginx/files/sites/opencraft-console/console.opencraft.com.conf

```diff
@@ -0,0 +1,57 @@
+server {
```

Comment:
@Agrendalath, this is confusing. Don't we have a separate configuration set up and in use for OCIM?